tests/trainer/optimization/test_optimizers.py

```diff
@@ -299,9 +299,9 @@ def configure_optimizers(self):
     assert trainer.lr_schedulers[1]["scheduler"]._step_count == expected_steps[1]
 
 
-@pytest.mark.parametrize("fn", ("validate", "test"))
-def test_init_optimizers_during_evaluation(tmpdir, fn):
-    """Test that optimizers is an empty list during evaluation."""
+@pytest.mark.parametrize("fn", ("validate", "test", "predict"))
+def test_init_optimizers_during_evaluation_and_predict(tmpdir, fn):
+    """Test that optimizers is an empty list during evaluation and predict."""
```

Comment:
```suggestion
    """Test that optimizers is an empty list during evaluation and prediction."""
```
grammar police on duty :))))